### PR TITLE
Removed code related to google sign in.

### PR DESCRIPTION
**Pull request by ahsanishaq721**
## Pull request (PR) checklist

Please check if your pull request fulfills the following requirements:

- [ ] The PR is submitted to the `main` branch.
- [ ] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) and my PR doesn't break the rules.
- [ ] I've read and understand the [Developer Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Guidelines.md).
- [ ] I confirm that I've run the code locally and everything works as expected.
- [ ] My PR includes only the necessary changes to fix the issue (i.e., no unnecessary files or lines of code are changed).
- [ ] 🎬 I've attached a **screen recording** of using the new code to the next paragraph (if applicable).

## What's changed?
Describe with a few bullets **what's new:**

- ...

- ...

## Risk factors

**What may go wrong if we merge your PR?**

- ...
- ...

**In what cases won't your code work?**

- ...
- ...

## Does this PR close any GitHub issues?

- Closes #{ISSUE_NUMBER}

## Troubleshooting CI failures ❌

Pull request checks failing? Read our [CI Troubleshooting guide](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/CI-Troubleshooting.md).

### CODE REVIEW

File: .idea/.name
Comment (ILIYANGERMANOV) [inline .idea/.name:1]: Revert that change, we need this file so the project name is correct

**Review by ILIYANGERMANOV — CHANGES_REQUESTED** [review]
Revert this accidentally deleted file. Can you look if we can remove more Google login related code? For example, unused libraries and stuff in Settings

**Comment by Ahsan721** [comment]
Ok, let me check. Have you checked either the code is working or not.

**Comment by ILIYANGERMANOV** [comment]
> Ok, let me check. Have you checked either the code is working or not.

I haven't QA the app, that's why we ask for screen recordings on PRs. Overall, the deleted code seems okay to remove

**Pull request closed without merging**